src/extension/uriRebaser.ts

```diff
@@ -88,10 +86,10 @@ export class UriRebaser {
     // Notes:
     // If 2 logs have the same uri, then likely the same (unless the uri is super short)
     // If 2 logs don't have the same uri, they can still potentially be the same match
-    public async translateLocalToArtifact(localUri: string): Promise<string> { // Future: Ret undefined when certain.
+    public async translateLocalToArtifact(localUri: Uri): Promise<string | undefined> {
```

Comment:
This can return undefined now - something to check. I can add back a `localUri.fsPath` fallback if that sounds right.